pegasus/sites.v3/code.org/public/css/page/dance-styles.scss

```diff
@@ -65,29 +65,6 @@
   }
 }
 
-// Featured artists
```

Comment:
I refactored these styles from the code.org/dance page to be reusable across that page and the new code.org/music page.